scripts/update-version.py

```diff
@@ -127,8 +127,8 @@ def update_main_readme(version, confirm):
 
     else:
         # validate if the provided version is valid.
-        # e.g. 7.15.0
-        pattern = "d.d{2}.d"
+        # e.g. 8.0.1
+        pattern = "d.d{21}.d"
```

Comment:
shouldn't the pattern look like this?
```suggestion
        pattern = "d.d{1}.d"
```

Reply:
it should, thnaks.